src/python/build_management/build_manager.py

```diff
@@ -984,6 +993,44 @@ def delete(self):
     raise BuildManagerException('Cannot delete system build.')
 
 
+class AuxiliaryBuild(Build):
+  """An additional build that accompanies some other fuzzinng build."""
+
+  def __init__(self,
+               base_build_dir,
+               revision,
+               build_url,
+               build_dir_name='revisions'):
```

Comment:
is it OK to use the same build dir name? will there be any clashes?

Reply:
I was thinking that `_get_build_directory` would take care of this and generate a different hash, but now I realize that it might become a problem if fuzzing and auxiliary builds would have the same filename. Thanks, will change.